bedrock/mozorg/templates/mozorg/rise25/landing.html

```diff
@@ -0,0 +1,243 @@
+      <div class="r25-c-category-body">
+        <h3 class="r25-c-category-title">Artists</h3>
+        <p class="r25-c-category-desc">Reimagining AI’s creative potential.</p>
+        <p class="r25-c-category-expand">
+          They transcend traditional AI applications, like synthesizing visuals
+          or using large language models. Their projects, whether interactive
+          websites, films, or digital media, challenge our perceptions and
+          demonstrate how AI can amplify and empower human creativity. Their work
+          provokes thought and offers fresh perspectives on the intersection of
+          AI and art.
+        </p>
+      </div>
+
+      <p class="r25-c-category-cta">
+        <a class="r25-c-button" href="https://mozilla.formstack.com/forms/rise25" rel="external noopener" target="_blank">Nominate</a>
+      </p>
+    </div>
+
+    <div id="entrepreneurs" class="r25-c-category r25-t-purple">
+      <div class="r25-c-category-body">
+        <h3 class="r25-c-category-title">Entrepreneurs</h3>
+        <p class="r25-c-category-desc">Fueling AI’s evolution with visionary ventures.</p>
+        <p class="r25-c-category-expand">
+          These daring individuals are transforming imaginative ideas into reality.
+          They’re crafting businesses and solutions with AI to meet societal needs,
+          improve everyday life and forge new technological paths. They embody
+          innovation, steering startups and projects with a commitment to ethical
+          standards, inclusiveness and enhancing human welfare through technology.
+        </p>
+      </div>
+
+      <p class="r25-c-category-cta">
+        <a class="r25-c-button" href="https://mozilla.formstack.com/forms/rise25" rel="external noopener" target="_blank">Nominate</a>
+      </p>
+    </div>
+
+    <div id="change-agents" class="r25-c-category r25-t-orange">
+      <div class="r25-c-category-body">
+        <h3 class="r25-c-category-title">Change Agents</h3>
+        <p class="r25-c-category-desc">Cultivating inclusive AI.</p>
+        <p class="r25-c-category-expand">
+          Challengers lead the way in diversifying AI, bringing varied community
+          voices into tech. They focus on inclusivity in AI development, ensuring
+          technology serves and represents everyone, especially those historically
+          excluded from the tech narrative. They are community leaders, corporate
+          leaders, activists and outside-the-box thinkers finding ways to amplify
+          the impacts of AI for marginalized communities. Their work fosters an
+          AI environment of equality and empowerment.
+        </p>
+      </div>
+
+      <p class="r25-c-category-cta">
+        <a class="r25-c-button" href="https://mozilla.formstack.com/forms/rise25" rel="external noopener" target="_blank">Nominate</a>
+      </p>
+    </div>
+  </section>
+
+  <section class="mzp-l-content mzp-t-content-md">
+    <div class="r25-blog">
```

Comment:
**Suggestion:** In mobile, the image is too small to see. Is it possible to also vertically align these two elements?

![Screenshot 2024-02-27 at 10 47 22 AM](https://github.com/mozilla/bedrock/assets/42309026/47e52eb1-e345-4f72-be19-2b1a35d8d934)